src/revpi-factory-reset.sh

```diff
@@ -16,15 +18,25 @@ export NEWT_COLORS='root=,black entry=white,black'
 while [ ! -r /home/pi/.revpi-factory-reset ] ; do
 	clear
 	msg="Please select the Product Type:"
-	ovl=$(whiptail --notags --title "PRODUCT TYPE" --menu "$msg" 0 0 0 \
-		compact "RevPi Compact" \
-		connect "RevPi Connect(+) / Connect S" \
-		connect-se "RevPi Connect SE" \
-		core "RevPi Core / Core 3(+) / Core S" \
-		flat "RevPi Flat" \
-		3>&1 1>&2 2>&3)
-	if [ "$?" == "1" ]; then
-		return
+	if cm1-detection; then
+		msg="Core CM1 autodetected)
+		ovl=$(whiptail --notags --title "PRODUCT TYPE" --menu "$msg" 0 0 0 \
+			core-cm1 "RevPi Core (autodetected)" \
```

Comment:
`core-cm1` is not listed in the `full_devicetype_list`

Reply:
Please see line 24